test_rules_scala.sh

```diff
@@ -16,13 +16,12 @@ $runner bazel build test/...
 #$runner bazel build "test/... --all_incompatible_changes"
 $runner bazel test test/...
 $runner bazel test third_party/...
-# UnusedDependencyChecker doesn't work with strict_java_deps
-$runner bazel build "--strict_java_deps=ERROR -- test/... -test:UnusedDependencyChecker"
-$runner bazel build "--extra_toolchains=//scala:minimal_direct_source_deps -- test/... -test:UnusedDependencyChecker"
-#$runner bazel build "--strict_java_deps=ERROR --all_incompatible_changes -- test/... -test:UnusedDependencyChecker"
-$runner bazel test "--strict_java_deps=ERROR -- test/... -test:UnusedDependencyChecker"
-$runner bazel test "--extra_toolchains=//scala:minimal_direct_source_deps -- test/... -test:UnusedDependencyChecker"
-$runner bazel build "test_expect_failure/missing_direct_deps/internal_deps/... --strict_java_deps=warn"
+$runner bazel build "--extra_toolchains=//test/toolchains:high_level_transitive_deps_strict_deps_error -- test/..."
+$runner bazel build "--extra_toolchains=//scala:minimal_direct_source_deps -- test/..."
+#$runner bazel build "--extra_toolchains=//test/toolchains:high_level_transitive_deps_strict_deps_error --all_incompatible_changes -- test/..."
+$runner bazel test "--extra_toolchains=//test/toolchains:high_level_transitive_deps_strict_deps_warn -- test/..."
```

Comment:
Pretty sure `high_level_transitive_deps_strict_deps_warn` should have been `high_level_transitive_deps_strict_deps_error` since the [original line](https://github.com/bazelbuild/rules_scala/pull/1033/files#diff-9e850e1e0893c9ea027f87c0b45dffabL23) had `ERROR`

Reply:
Oops will fix